gqlerrors/decode.go

```diff
@@ -0,0 +1,38 @@
+package gqlerrors
+
+import (
+	"reflect"
+
+	"github.com/gofrs/uuid"
+	"github.com/mitchellh/mapstructure"
+)
+
+// Decode an objects
+func Decode(input interface{}, output interface{}) error {
+	decoder, err := mapstructure.NewDecoder(&mapstructure.DecoderConfig{
+		TagName: "json",
+		Result:  output,
+		DecodeHook: mapstructure.ComposeDecodeHookFunc(
+			toUUIDHookFunc,
```

Comment:
We don't have any UUID in the gqlerrors.FormattedError, so we may don't need this overhead. Here's a [reference](https://github.com/kelseyhightower/nocode).